pilot/pkg/security/authz/builder/builder_test.go

```diff
@@ -51,19 +52,22 @@ func TestGenerator_GenerateHTTP(t *testing.T) {
 		name        string
 		tdBundle    trustdomain.Bundle
 		isVersion14 bool
+		enableRegex bool
 		input       string
 		want        []string
 	}{
 		{
 			name:        "path14",
 			input:       "path14-in.yaml",
 			isVersion14: true,
+			enableRegex: true,
```

Comment:
Don't need regex for path14, same for path15, it should be covered in all-fields-regex-enabled

Reply:
But "all fields in" are only testing regex matching after version 15, do we need to keep it for path-14?